### PR TITLE
Web consistency #88 #87 #65

### PR DESCRIPTION
**Pull request by JasminFarhi**
This covers
headers consistency 
Mesh page bugs
holacracy principals added

### CODE REVIEW

File: careers.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <head>
   <meta charset="utf-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <title>dltx.io | THE MESH</title>
+  <title>dltx.io | CAREERS</title>
```
Comment (Mitchell139) [inline careers.html:12]:
```suggestion
  <title>dltx.io | Careers</title>
```

File: clients.html [hunk 2]
```diff
@@ -87,7 +93,7 @@ <h2 class="to-animate">Development and/or Advisory Services</h2>
                             <div class="row">
                               <div class="core col-sm-6">
                                 <div class="fh5co-post to-animate">
-                                  <p>
+                                  <p style="color:#ffffff;">
```
Comment (Mitchell139) [inline clients.html:96]: No to inline styling, especially in this situation!

File: themesh.html [hunk 3]
```diff
@@ -50,30 +50,40 @@
...
+    </div>
+  </section>
+
+  <section id="fh5co-services" data-section="services">
+    <div class="fh5co-services">
+      <div class="container">
+        <div class="row">
+          <!-- HERO SECTION PHILOSOPHY -->
+          <div class="col-md-12 section-heading text-center">
+            <h3 class="to-animate">The DLTx mesh a <a href="https://www.holacracy.org/explore">holacracy</a> based organisation, which is a system for managing a company where there are no assigned roles and employees have the flexibility to take on various tasks and move between teams freely. The organizational structure of a holacracy is rather flat, with there being little hierarchy.</h3>
```
Comment (Mitchell139) [inline themesh.html:80]: That sentence doesn't read properly. What is the point you are trying to get across?

File: venture-studio.html [hunk 4]
```diff
@@ -76,7 +77,7 @@ <h1 class="to-animate">dltx.io</h1>
         <div class="row">
           <!-- HERO SECTION PHILOSOPHY -->
           <div class="col-md-12 section-heading text-center">
-            <h2 class="to-animate">Venture Studio</h2>
+            <h2 class="to-animate">The Studio</h2>
```
Comment (Mitchell139) [inline venture-studio.html:80]: Did you mean to change that?
```suggestion
            <h2 class="to-animate">Venture Studio</h2>
```

**Review by Mitchell139 — CHANGES_REQUESTED** [review]

**Review by bitcoinbrisbane — APPROVED** [review]

File: clients.html [hunk 5]
```diff
@@ -20,7 +20,7 @@
   <meta property="og:url" content="" />
   <meta property="og:site_name" content="" />
   <meta property="og:description" content="" />
-  <meta name="twitter:title" content="dltxio" />
+  <meta name="twitter:title" content="" />
```
Comment (bitcoinbrisbane) [inline clients.html:23]: I put this in :(

**Comment by bitcoinbrisbane** [comment]
@JasminFarhi can you post a screen shot?  Not too keen on hacking with inline styling.  Its probably just missing a class on the div

**Pull request merged**